rosdep/python.yaml

```diff
@@ -8008,6 +8008,11 @@ python3-onnxruntime-pip:
   ubuntu:
     pip:
       packages: [onnxruntime]
+python3-open3d:
+  debian:
+    bullseye: [python3-open3d]
+  ubuntu:
+    jammy: [python3-open3d]
```

Comment:
If we assume this is going to be released into future distributions, wouldn't it be more future-proofed to label this with the name as `'*'` and then null out any active distributions its not available in?

Reply:
Good idea, I've pushed https://github.com/ros/rosdistro/pull/36052/commits/e2f538907999db6a3265901ad593f4dcbba65532 so that `python3-open3d` is available for only the Ubuntu and Debian releases that ship with it